### ISSUE TITLE
Airport documentation needs an update...

### ISSUE THREAD

**Issue by panther2** [openscope/openscope#430]
... to include #310 / #425

**Comment by erikquinn** [comment]
This is included in the spawn pattern readme actually. It is linked to in the airport format document. We could add it both places though I guess?

**Comment by panther2** [comment]
That would be fine. Personally - while creating my first Airport.json -  I would prefer to see it right in the airport-format.md when coding the procedures, as I would not think about looking into the spawn pattern document at that point.

**Comment by n8rzz** [comment]
@panther2 so the rationale there is that the spawnPatterns are rather complex and deserved their own document.  whereas the airport-format doc itself is just an api specification.  The way I worked it out in my head is that an author would create the airport and all the supporting fixes, then move on to create spawnPatterns.

Whatever makes it easiest for airport authors, thats the main goal here.  I don't know, maybe it does make sense to have them all in one spot?

**Comment by panther2** [comment]
@n8rzz 
Yes agreed. From the view of an airport maker (apprentice) I definitely need the information in the airport-format.md because I come to coding the procedures **before** even realizing what spawn patterns might be. When it (as follow-up to the procedures) comes to coding the spawn-patterns into the airport file, I will be happy to study the spawnPatterns document.

But maybe my coding sequence (following the airport-format.md chronologically) is sub-optimal?
In that case I certainly lack the experience...

**Comment by n8rzz** [comment]
I think that would be the way to go, top down.  Before you can even think about writing the spawnPatterns you need to have the fixes and procedures defined.  the spawnPatterns themselves merely describe how aircraft should enter the sim and what characteristics each aircraft should have when they enter.

**Comment by erikquinn** [comment]
What probably needs to happen is the writing of _routes_ needs its own section describing how dot notation works, plus all the special characters (`_ / @ / #`) and how to use them, etc. This could be either a section of the airport file, or a separate document, linked to from both the airport-format.md _AND_ spawn-pattern-readme.md.
